docs/spatial_dbms.rst

```diff
@@ -30,21 +30,24 @@ Supporting installation of spatial data into `Postgres DBMS`.
   This is done by using either `Postgres CLI` or `GUI(PgAdmin)` and run
   **For psql CLI**
 
-.. code-block:: sh
+.. note:: New versions of Postgres use `CREATE EXTENSION postgis_raster;`
```

Comment:
I'd recommend separate instructions by version to make this clear. So one for after the change and one for before the change. Alternatively I think it would be OK to have a minimum postgres version that we support that is one after this change was made.